config/200-clusterrole.yaml

```diff
@@ -36,3 +36,6 @@ rules:
   - apiGroups: ["discovery.knative.dev"]
     resources: ["*"]
     verbs: ["get", "list", "create", "update", "delete", "deletecollection", "patch", "watch"]
+  - apiGroups: ["rbac.authorization.k8s.io"]
+    resources: ["clusterroles"]
+    verbs: ["get"]
```

Comment:
you also might need list and watch here

Reply:
yep, `list` was required to provide an alternative to specifying the exact cluster role.
Not using `watch` at the moment, but figure it will be needed to optimize reconciling loop.